src/components/TicketModal/TicketModal.vue

```diff
@@ -2,175 +2,277 @@
+          membersPage.value
+        );
+        selectedUser.value = "None";
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        ticketObj.value.assigned_user = -1;
+      } catch (error) {
+        alert(error);
+      }
     };
-    const closeModal = () => {
-      context.emit("close");
+    const getStatuses = async () => {
+      try {
+        statuses.value = Object.values(
+          await listStatuses(selectedTeam.value.id)
+        );
+        selectedStatus.value.title = statuses.value[0].title;
+        selectedStatus.value.color = statuses.value[0].color;
+        selectedStatus.value.id = statuses.value[0].id;
+      } catch (error) {
+        alert(error);
+      }
+    };
+    const statusSelected = (display: string, item: StatusRecordOutput) => {
+      selectedStatus.value.title = display;
+      selectedStatus.value.color = item.color;
+      selectedStatus.value.id = item.id;
+    };
+    const userSelected = (display: string, item: MemberRecord) => {
+      selectedUser.value = display;
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      ticketObj.value.assigned_user = item.id;
+    };
+    const teamSelected = async (display: string, item: ReadTeamRecord) => {
+      if (selectedTeam.value.id !== item.id) {
+        selectedTeam.value.name = display;
+        selectedTeam.value.id = item.id;
+        await getTags();
+        await getMembers();
+        await getStatuses();
+      }
+    };
+    const tagSelected = (display: string, item: TagRecord) => {
+      selectedTag.value.title = display;
+      selectedTagId.value[0] = item.id;
+    };
+    const setTitle = (item: string) => {
+      ticketObj.value.title = item;
+      shouldShowPencil.value = true;
     };
-    const saveChanges = () => {
+    const closeModal = () => {
       context.emit("close");
     };
-    const setTicketData = () => {
-      if (props.ticketRecord) {
-        const ticket = props.ticketRecord;
-        ticketStatus.value = ticket.status?.title || "";
-        ticketUser.value = ticket.assigned_user?.username || "";
+    const saveChanges = async () => {
+      const ticket: WriteTicketRecord = {
+        title: ticketObj.value.title,
+        status: selectedStatus.value.id
+      };
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      if (ticketObj.value.due_date) {
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        ticket.due_date = ticketObj.value.due_date;
+      }
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      if (ticketObj.value.assigned_user !== -1) {
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        ticket.assigned_user = ticketObj.value.assigned_user;
+      }
+      if (ticketObj.value.description !== "") {
+        ticket.description = ticketObj.value.description;
+      }
+      if (selectedTagId.value[0] !== -1) {
+        // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
you can do `/* eslint-disable @typescript-eslint/camelcase */` at the top of the `<script>` section to avoid duplicating these comments.

Reply:
fixed